src/css/pin.css

```diff
@@ -0,0 +1,28 @@
+@layer components {
+    .bcc-pin {
+        @apply inline-block rounded-full p-[.3em] shadow-md text-xs;
+    }
+
+    .bcc-pin .bcc-pin-icon {
+        @apply h-[1.2em] w-[1.2em]
```

Comment:
Since we only have the one size from the design now, this could/should be `h-4 w-4`.